setup.cfg

```
@@ -29,7 +29,7 @@ project_urls =
 [options]
 packages = find:
 install_requires =
-    attrs>=19,<21
+    attrs>=19
```

```suggestion
    attrs>=19,<22
```

cheers! I still want to pin this for future releases though; to guard against any nasty surprises later on